### ISSUE TITLE
Training loss not convergence

### ISSUE THREAD

**Issue by fire717** [PyTorchLightning/pytorch-lightning#4862]
## 🐛 Bug

Running the example code below,  at my pc , the loss not change:

`Epoch 29:  43%|████████████████████████▋                                 | 800/1875 [00:04<00:05, 189.48it/s, loss=1.483, v_num=87]`

as u see, even at epoch29, the loss is around 1.5.

code:
`import os

import torch
from torch.nn import functional as F
from torch.utils.data import DataLoader
from torchvision.datasets import MNIST
from torchvision import transforms
import pytorch_lightning as pl

class MNISTModel(pl.LightningModule):

    def __init__(self):
        super(MNISTModel, self).__init__()
        self.l1 = torch.nn.Linear(28 * 28, 10)

    def forward(self, x):
        return torch.relu(self.l1(x.view(x.size(0), -1)))

    def training_step(self, batch, batch_nb):
        x, y = batch
        loss = F.cross_entropy(self(x), y)
        tensorboard_logs = {'train_loss': loss}
        return {'loss': loss, 'log': tensorboard_logs}

    def configure_optimizers(self):
        return torch.optim.Adam(self.parameters(), lr=0.02)

train_loader = DataLoader(MNIST(os.getcwd(), train=True, download=True, transform=transforms.ToTensor()), batch_size=32)

mnist_model = MNISTModel()
trainer = pl.Trainer(gpus=1, progress_bar_refresh_rate=20)    
trainer.fit(mnist_model, train_loader)  `

## Please reproduce using [the BoringModel and post here]

Here comes the strange thing, with **same code**, I running at the colab,
(https://colab.research.google.com/drive/1rPmzscbE4tCo7k8pj7FQJxck3JcM-Zx8?usp=sharing)

Got this
`Epoch 1: 77%
1440/1875 [00:06<00:01, 230.34it/s, loss=0.669, v_num=0]`

Even at epoch 1, I got loss = 0.669!

### To Reproduce
Colab:https://colab.research.google.com/drive/1rPmzscbE4tCo7k8pj7FQJxck3JcM-Zx8?usp=sharing
my local pc maybe you cannot Reach

### Expected behavior

Colab and my pc get Same behavior , in other words,  my pc loss should convergence.

### Environment

 - PyTorch Version (e.g., 1.0): 1.6.0
 - OS (e.g., Linux): Linux
 - How you installed PyTorch (`conda`, `pip`, source):   pip
 - Python version: 3.7.4
 - CUDA/cuDNN version: 10.1
 - GPU models and configuration:   Gforce1060 (1080ti also get loss around 1.5 all the time.)
 - Any other relevant information:

**Comment by fire717** [comment]
change to pytoch-lightning 1.0.0  there is no problem. (use 1.0.8 before)
